packages/upload-api/src/index/add.js

```diff
@@ -87,3 +96,17 @@ const assertAllocated = async (context, space, digest, errorName) => {
     )
   return ok({})
 }
+
+/**
+ * @param {API.ClaimsClientContext} ctx
+ * @param {{ content: API.UnknownLink, index: API.CARLink }} params
+ */
+const publishIndexClaim = async (ctx, { content, index }) => {
+  const { invocationConfig, connection } = ctx.claimsService
+  const { issuer, audience, with: resource, proofs } = invocationConfig
+  const expiration = Infinity
+  const nb = { content, index }
+  const conf = { issuer, audience, with: resource, nb, expiration, proofs }
```

Comment:
nit: I would add this in the `invoke` with new lines per property to be more visible